mypy_django_plugin/lib/helpers.py

```diff
@@ -175,14 +175,22 @@ def get_call_argument_by_name(ctx: Union[FunctionContext, MethodContext], name:
     Return the expression for the specific argument.
     This helper should only be used with non-star arguments.
     """
-    if name not in ctx.callee_arg_names:
-        return None
-    idx = ctx.callee_arg_names.index(name)
-    args = ctx.args[idx]
-    if len(args) != 1:
-        # Either an error or no value passed.
-        return None
-    return args[0]
+    # first check for named arg on function definition
+    if name in ctx.callee_arg_names:
+        idx = ctx.callee_arg_names.index(name)
+        args = ctx.args[idx]
+        if len(args) != 1:
+            # Either an error or no value passed.
+            return None
+        return args[0]
+
+    # check for named arg in function call keyword args
```

Comment:
Would be helpful to have more detailed docstring or code comments explaining what's going on here, why there are 2 ways to look up an argument by name and when each is triggered.

If I didn't already know the context explained in this the PR description, I probably would be confused reading this function.